hbase-protocol-shaded/src/main/protobuf/server/master/RegionServerStatus.proto

```diff
@@ -97,6 +97,9 @@ message RegionStateTransition {
   optional uint64 open_seq_num = 3;
 
   repeated int64 proc_id = 4;
+
+  // Master active time as fencing token
```

Comment:
There are some duplicate file under hbase-protocol module also? Do we need to update there as well ?